### ISSUE TITLE
Search: No results with comma, but results with space

### ISSUE THREAD

**Issue by lucidBrot** [AdventureLookup/AdventureLookup#280]
Please excuse if I just misunderstood something.  

The [site](https://www.adventurelookup.com/adventures) states that the search terms will be split at `,`, so I entered `sandbox, roleplay` - this gives 0 results. `sandbox,roleplay` also gives 0 results. `sandbox roleplay` however gives 30 results.  

`goblins, mountains`, the suggested search, gives 42 results. `goblins mountains` gives 520 results.  

So either this is a bug or at least it needs to be made clearer why this happens if this is in fact intended behaviour.

**Comment by cmfcmf** [comment]
I know this is a late response, but I finally found the time to look into the ElasticSearch integration more closely.

This is the expected (albeit confusing) behavior. By default, terms are combined with OR (e.g., searching for `sandbox roleplay` returns adventures that contain both terms or just one of the terms). Searching for `sandbox, roleplay` causes the terms to be ANDed, resulting only in adventures which contain both terms.

Since this is behavior is quite confusing, I changed it in #308, which will soon go live on adventurelookup.com. It then works like this:
- searching for 'sandbox roleplay' results in adventures that contain both terms
- searching for 'sandbox OR roleplay' results in adventures that contain both terms, then those that contain just one of the terms.
